coordinator/tests/resources/test-resources.ts

```diff
@@ -4,26 +4,32 @@ import * as path from "path"
 import {ParentPrescription} from "../../src/services/hl7-v3-prescriptions";
 import {Bundle} from "../../src/services/fhir-resources";
 
+/**
```

Comment:
I have a bit of an anti-comment bias, but if these are just for tests, could we write functions to return these instead of commenting the code blocks? `createRepeatPrescription()` / `createAcutePrescription()`

Reply:
Can just get rid of the comments if you like, they don't need to be there. Just thought it neatened things up

Reply:
Discussed - changed to export objects which group related test resources together.